package.json

```diff
@@ -183,6 +193,8 @@
     "lint-fix": "eslint  --max-warnings 0 --fix .",
     "postinstall": "patch-package && jetify",
     "validate": "npm ls",
-    "start": "make dev"
+    "start": "make dev",
+    "e2e-tests": "wdio run wdio.conf.js",
+    "e2e-test-reports": "npx wdio run ./wdio.conf.js && allure generate allure-results/ --clean && allure open"
```

Comment:
Let's make sure reports are always created, but don't try to open them because this will run on a CI 99% of the time.